tests/python/pytorch/graphbolt/test_integration.py

```diff
@@ -64,19 +64,19 @@ def test_integration_link_prediction():
                         [3, 2],
                         [3, 2],
                         [3, 3],
-                        [5, 0],
-                        [5, 0],
-                        [3, 3],
+                        [5, 5],
```

Comment:
why the expected results are changed?

Reply:
I guess torch.utils.data.DataLoader does things that we don't know about.

Reply:
The code is here, it is quite long so we need good justification to keep deriving from it. https://github.com/pytorch/pytorch/blob/main/torch/utils/data/dataloader.py

Reply:
This is weird, it seems that the seed is respected in a different way.